src/components/NavBar/NavBar.scss

```diff
@@ -1,5 +1,159 @@
+  font-weight: 600;
+
+  @media (max-width: 500px) {
+    display: flex;
+    flex-direction: column;
+  }
+
+  nav {
+    z-index: 103 !important;
+  }
+
+  .logo {
+    font-size: 30px;
+    display: flex;
+    justify-content: flex-end;
+    // padding-left: 8.4rem;
+    padding-top: 0.2rem;
+    width: 15.1%;
+
+    @media (max-width: 600px) {
+      display: flex;
+      justify-content: center;
+      padding: 0;
+    }
+
+    @media (max-width: 500px) {
+      display: flex;
+      justify-content: center;
+      padding: 0;
+    }
+
+    a {
+      color: rgba(255, 255, 255, 1);
+      text-decoration: none;
+
+      span {
+        color: #e84c3d !important;
+      }
+    }
+  }
+
+  .nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    // border: 1px solid red;
+    width: 60%;
+    font-size: 14px;
+
+    @media (max-width: 849px){
+      display: flex;
+      flex-direction: row;
+      justify-content: space-evenly;
+      width: 60%;
+    }
+
+    @media (max-width: 600px) {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-evenly;
+    }
+
+    @media (max-width: 500px) {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-evenly;
+    }
+
+    .links {
+      // border: 1px solid green;
+      width: 60%;
+      display: flex;
+      justify-content: space-between;
+
+      @media (max-width: 849px){
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-evenly;
+        width: 80%;
+      }
+
+      @media (max-width: 600px) {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-evenly;
+        width: 75%;
+      }
+
+      @media (max-width: 500px) {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: space-evenly;
+      }
+
+      @media (max-width: 414px) {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: space-evenly;
+        
+      }
+
+      @media (max-width: 375px) {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: space-evenly;
+        
+      }
+
+      @media (max-width: 320px) {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: space-evenly;
+        
+      }
+
+      a {
+        color: rgba(255, 255, 255, 1);
+        padding: 0.12rem 0;
+      }
+
+      i {
+        color: rgba(255, 255, 255, 1);
+        padding: 0 0 0.6rem 0;
+      }
+    }
+
+    button {
+      color: rgb(232, 76, 61);
+      background-color: rgba(255, 255, 255, 1);
+      height: 2rem;
+      padding-bottom: 1.6rem;
+    }
+  }
+}
+
+
+// orig code
+
+/*
+
+@import url('https://fonts.googleapis.com/css?family=Raleway:600&display=swap');
+
```

Comment:
feel free to erase old code if it no longer serves us. Marie Kondo would ask if it brings us joy. I think no.